src/internal/common/limiters/sliding_window.go

```diff
@@ -107,8 +115,42 @@ func (s *slidingWindow) Shutdown() {
 	})
 }
 
+// Reset resets the limiter to its initial state. Any pending requests will be
+// cancelled. Reset should only be called when the limiter is not in use.
+// No need to shutdown slide goroutine. prev and curr
```

Comment:
nit: hanging comment. Also since this is more about the implementation it should be moved into the function body itself so it doesn't show up in godoc (unless it's removed completely if it's not needed anymore)

Reply:
Ah thanks! I don't know how this comment ended here :O